src/components/inquiry_artwork_brick/details.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+import colors from "../../assets/colors"
+import TextLink from "../text_link"
+
+export interface DetailsProps extends React.HTMLProps<ArtworkDetails> {
+  artwork: any
+}
+
+const DetailsContainer = styled.div`
+  height: 100px;
+  margin-top: 10px;
+`
+
+export class ArtworkDetails extends React.Component<DetailsProps, null> {
+  artistLine() {
+    const { cultural_maker, artists } = this.props.artwork
+
+    if (cultural_maker) {
+      return <div><strong>{cultural_maker}</strong></div>
+    } else if (artists && artists.length) {
+      let artistLine = artists
```

Comment:
Use `const` instead of `let` if the variable is never going to change. I’m surprised the linter didn’t catch this 🤔